recipes/README.md

```diff
@@ -0,0 +1,118 @@
+# Granite Code Cookbook
+
+The "Recipes" in the Granite Code Cookbook showcase the capabilities of
+the Granite Code models.
+
+## Contents
+
+The Granite Code Cookbook is still in early development.
```

Comment:
Was expecting to see a change to this line

Reply:
@adampingel ok fixed.